.editorconfig

```diff
@@ -0,0 +1,10 @@
+root=true
+
+[*]
+end_of_line = lf
+indent_style = space
```

Comment:
This could cause wars! Let me see what the defaults are in VS for each file type